src/amo/components/InstallWarning/index.js

```diff
@@ -88,17 +96,19 @@ export class InstallWarningBase extends React.Component<InternalProps> {
   };
 
   componentDidMount() {
-    const { _log, _tracking, variant } = this.props;
+    const { _log, _tracking, clientApp, variant } = this.props;
 
     if (!variant) {
       _log.debug(`No variant set for experiment "${EXPERIMENT_ID}"`);
       return;
     }
 
-    _tracking.setDimension({
-      dimension: INSTALL_WARNING_EXPERIMENT_DIMENSION,
-      value: variant,
-    });
+    if (clientApp !== CLIENT_APP_ANDROID) {
```

Comment:
Or maybe we should force `CLIENT_APP_FIREFOX` instead? I know we only have two client apps nowadays but..